### PR TITLE
add new targets and improve existing

### PR DESCRIPTION
**Pull request by Kaiowarez**
NBD-INFINITY folder name - NBDINFINITY ; - in name causes an unexpected warning
iflight target still needs to be confirmed not sure if its working, but compiles properly

### CODE REVIEW

File: src/main/target/IFF7TWING/target.c [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    DEF_TIM(TIM8,  CH1, PC6,  TIM_USE_MOTOR,  0, 0), // ESC 2
+    DEF_TIM(TIM8,  CH4, PC9,  TIM_USE_MOTOR,  0, 0), // ESC 3
+    DEF_TIM(TIM8,  CH2, PC7,  TIM_USE_MOTOR,  0, 1), // ESC 4
+    DEF_TIM(TIM4,  CH1, PB6,  TIM_USE_MOTOR,  0, 0), // ESC 5 
+    DEF_TIM(TIM4,  CH2, PB7,  TIM_USE_MOTOR,  0, 0), // ESC 6
+    DEF_TIM(TIM3,  CH4, PB1,  TIM_USE_MOTOR,  0, 0), // ESC 7
+    DEF_TIM(TIM3,  CH3, PB0,  TIM_USE_MOTOR,  0, 0), // ESC 8
+
+    DEF_TIM(TIM2,  CH2, PA1,  TIM_USE_LED,    0, 0), // LED Strip
+};
```
Comment (Quick-Flash) [inline src/main/target/IFF7TWING/target.c]: Can you get this Iflight target tested before we merge it? Once you can confirm it works I'd love to see it merged.

**Comment by Kaiowarez** [comment]
Cools will do, white from the French guys is testing it, just waiting in
his feedback

**Comment by Quick-Flash** [comment]
Let me know when you can confirm it works and i'll merge this :)

**Comment by Kaiowarez** [comment]
chad getting all the targets tested now

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=icon>
Virus-free.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=link>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by Quick-Flash** [comment]
> chad getting all the targets tested now

Brilliant once we can confirm let me know and I'll merge this.

**Comment by Kaiowarez** [comment]
Will do

**Comment by nerdCopter** [comment]
Double PR #115 & #131 is confusing, imo.
Does this one need to be closed, and the other one live?  
Should the PR's be for one target at a time? (it's easy to create branches & PR's)
PR's should be "atomic".  i.e. One feature/bugfix

**Pull request closed without merging**